src/frontends/pytorch/src/utils.cpp

```diff
@@ -232,7 +232,7 @@ OutputVector make_framework_node(NodeContext* context) {
                                           context->get_output_size() - num_body_outs + num_skip_body_outputs);
     fw_node->set_friendly_name(context->get_op_type());
     for (size_t i = 0; i < bodies.size(); ++i) {
-        fw_node->set_function(i, bodies[i]);
+        fw_node->set_function(static_cast<int>(i), bodies[i]);
```

Comment:
may be it is a problem of API `set_function`? Why do we use `int` type for body index. `size_t` is better.

Reply:
if it is not FE code, I am fine then. I thought this is PT FE FW Node API:)